args4j/src/org/kohsuke/args4j/CmdLineParser.java

```diff
@@ -548,7 +548,7 @@ public void parseArgument(final String... args) throws CmdLineException {
      * by the @ sequences, split around the line breaks.
      * @throws CmdLineException 
      */
-    private String[] expandAtFiles(String args[]) throws CmdLineException {
+    protected String[] expandAtFiles(String args[]) throws CmdLineException {
         List<String> result = new ArrayList<String>();
```

Comment:
Just in case. `args4j` has a habit of having `private` methods that result in me having to maintain my own fork. I wouldn't expect anyone to `extend CmdLineParser` unless then know what they're doing.